github-pull-requests.go

```diff
@@ -39,82 +38,64 @@ type GitPRs struct {
 	ReviewState []ReviewState
 }
 
-// PRIssues - holds all PRs
-type PRIssues struct {
-	ID      int    `json:"id"`
-	HTMLURL string `json:"html_url"`
-	Number  int    `json:"number"`
-	State   string `json:"state"`
-	Title   string `json:"title"`
-	User    struct {
-		Login string `json:"login"`
-	} `json:"user"`
-	CreatedAt time.Time `json:"created_at"`
-	UpdatedAt time.Time `json:"updated_at"`
-	Assignee  struct {
-		Login string `json:"login"`
-	} `json:"assignee"`
-	Assignees []struct {
-		Login string `json:"login"`
-	} `json:"assignees"`
-	Head struct {
-		User struct {
-			Login string `json:"login"`
-		} `json:"user"`
-		Repo struct {
-			Name string `json:"name"`
-		} `json:"repo"`
-	} `json:"head"`
-}
-
-var pRequests []GitPRs
-
-func populatePRs(rName string, url string) {
-	pullURL := url + token
-	fmt.Println("Fetching from....", pullURL)
-	resp, err := http.Get(pullURL)
-	exitOnErr(err)
-
-	defer resp.Body.Close()
-	htmlData, err := ioutil.ReadAll(resp.Body)
-	exitOnErr(err)
+// Retrieve the relevant pull request data from an owner/repo.
+func getPullRequests(owner string, repo string) (pullRequests []GitPR, err error) {
+	prs, _, err := buzzClient.PullRequests.List(ctx, owner, repo, nil)
+	if err != nil {
+		return nil, err
+	}
 
-	pIssues := []PRIssues{}
-	json.Unmarshal(htmlData, &pIssues)
+	pullRequests = []GitPR{}
 
-	for _, elem := range pIssues {
-		eachPRIssue := GitPRs{}
-		eachPRIssue.Number = elem.Number
-		eachPRIssue.Title = elem.Title
-		eachPRIssue.Sender = elem.Head.User.Login
+	for _, elem := range prs {
+		pullRequest := GitPR{}
+		pullRequest.Number = *elem.Number
```

Comment:
[`github.PullRequest`]'s members are pointers, not values, and we're reusing their code instead of writing our own so we just need to adopt that to switch